_includes/components/homepage/Revenue/index.jsx

```diff
@@ -10,6 +10,9 @@ function Revenue() {
       const component = componentsList[i];
       const rawItems = JSON.parse(component.getAttribute('data-info')).data;
       const link = component.getAttribute('data-link');
+      const year = component.getAttribute('data-year') || '2017-18';
```

Comment:
I'm concerned about this fallback - why do we need it?

Reply:
If there is a problem and the year does not pull through that it atleast uses the most up to date year, instead of `/undefined/search-results?search=.....`

Reply:
Although the fallback in the dynamic budget portal is for known missing data which we'll follow up with after budget day.